docs/docs-content/clusters/data-center/nutanix/install-pcg.md

```diff
@@ -0,0 +1,217 @@
+
+- A Nutanix subnet created in Nutanix Prism Central that will be assigned to the virtual machines (VMs) deployed in the Kubernetes cluster.
+
+- A Nutanix cloud registered with Palette. For more information, review [Register Nutanix Cloud](register-nutanix-cloud.md).
+
+- A Nutanix Cluster API (CAPI) OS image. For guidance on creating the image, refer to [Building CAPI Images for Nutanix Cloud Platform](https://image-builder.sigs.k8s.io/capi/providers/nutanix.html#building-capi-images-for-nutanix-cloud-platform-ncp).
+
+
+
+
+## Install PCG
+
+Use the following steps to install the PCG in your Kubernetes workload cluster.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Tenant Settings**.
+
+3. Next, on the **Tenant Settings Menu**, select **Private Cloud Gateways** and click on **Add New Private Cloud Gateway**.
+
+4. Select **Self Hosted** in the next window that Palette displays.
+
+5. Provide a name for the PCG and use the **drop-down Menu** to select Nutanix as the cloud type. Click **Confirm** to continue. You will be redirected to the Private Cloud Gateway Overview page. 
+
+6. To install the Palette agent, copy the kubectl commands from the slide-out panel and execute them against your workload cluster.
+
+7. Close the slide-out panel when you have copied both commands. The PCG Overview page **Cluster Status** field will display **Pending** while the PCG is deploying. The deployment is complete when the **Cluster Status** field displays the status **Running**.  
+
+
+## Validate
+
+When deployed, the PCG registers itself with Palette. Use the steps below to verify if the PCG registration is successful.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+
+2. Navigate to the **left Main Menu** and select **Tenant Settings**.
+
+
+3. Next, on the **Tenant Settings Menu**, select **Private Cloud Gateways**.
+
+
+4. Locate the PCG and verify it is installed and in the **Running** state. With the PCG successfully deployed in your Kubernetes workload cluster, you can delete the kind cluster that was used to bootstrap the workload cluster. 
+
+  ```bash
+  kind delete cluster --name pcg-pilot
+  ```
+
+## Set Up Environment and Deploy Cluster
+
+This section describes one possible method for creating a Nutanix cluster to deploy your PCG using the process described in the [Nutanix Getting Started](https://opendocs.nutanix.com/capx/v1.1.x/getting_started/) resource and the [Common Prerequisites](https://cluster-api.sigs.k8s.io/user/quick-start#common-prerequisites) it specifies.
```

Comment:
```suggestion
This section describes one possible method for creating a Kubernetes cluster that will be used to deploy your PCG using the process described in the [Nutanix Getting Started](https://opendocs.nutanix.com/capx/v1.1.x/getting_started/) resource and the [Common Prerequisites](https://cluster-api.sigs.k8s.io/user/quick-start#common-prerequisites) it specifies.
```

Can we have this highlighted, maybe as an infotip paragraph?

Reply:
There is no need to highlight this, it's the entry paragraph to the section.

Reply:
:white_check_mark: